src/dryad/format/endf/createProjectileIdentifier.hpp

```diff
@@ -0,0 +1,46 @@
+#ifndef NJOY_DRYAD_FORMAT_ENDF_CREATEPROJECTILEID
+#define NJOY_DRYAD_FORMAT_ENDF_CREATEPROJECTILEID
+
+// system includes
+#include <vector>
+
+// other includes
+#include "tools/Log.hpp"
+#include "dryad/ParticleID.hpp"
+
+namespace njoy {
+namespace dryad {
+namespace format {
+namespace endf {
+
+  /**
+   *  @brief Create the projectile identifier based of the sublibrary value
+   */
+  ParticleID createProjectileIdentifier( int sublibrary ) {
+
+    switch ( sublibrary ) {
+
+      case     0 : return ParticleID( "g" );
+      case     3 : return ParticleID( "g" );
+      case    10 : return ParticleID( "n" );
+      case   113 : return ParticleID( "e-" );
```

Comment:
Is there a separate identifier for `e+`?

Reply:
Not as far as I know. I'm not aware of a positron sublibrary.

Reply:
Hmm. Reading through the documents, this is very much electron-specific and should not be ambiguated with positrons. So that answers my second question.